Bottom: 04edbd9d3f63cc91346e1ef85f0043e728dd71c4
Top:    86b751fb4ad1de01d62548f41cccada2cf8fe256
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 18:40:49 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 72b6903..74f26d7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (hidGC gc, ElementType *Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && hid_draw_is_gui (hid_draw)) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& hid_draw_is_gui (hid_draw)*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
